beamer/tests/conftest.py

```diff
@@ -178,8 +178,8 @@ def config(request_manager, fill_manager, token, token_list):
     account = eth_account.Account.from_key(_LOCAL_ACCOUNT.private_key)
     token.mint(account.address, 300)
     url = URL(ape.config.provider.uri)
-    rpc_urls = {"l1": URL(url), "l2a": URL(url), "l2b": URL(url)}
-    poll_period_per_chain = {"l1": 2.0}
+    poll_period_per_chain = {"l2a": 1.1}
```

Comment:
Why are we making L2 poll period bigger than the one for L1?

Reply:
there is no l1 eventmonitor anyway ? so i was thinking we missed this in previous pr imo. that is why i have now l2a and previously tests were not failing as i explained but with 2 secs normally timeouts happening inn tests so i still wanted to have difference so changed it to become 1.1

Reply:
> there is no l1 eventmonitor anyway ?

What do you mean?
This conftest and the config here is common to all tests, since it is at the top of the pytest hierarchy, no?
Wherever we have an agent, we will have event monitors which would be affected by this, assuming they don't customise the poll period.

> so i was thinking we missed this in previous pr imo.

Which PR?

> that is why i have now l2a and previously tests were not failing as i explained but with 2 secs normally timeouts happening inn tests

Could you please expand on this part, please? 

Also, if we really need to specify a poll period, why not go into the other direction and use, say, `0.9` or `0.8`?

Reply:
https://github.com/beamer-bridge/beamer/pull/1596 this one. so the thing here if you look at the agent.py. if chain_name is l1 then it continues and actually there is no event monitor right? so that is why adding poll period 2.0 here in conftest for the l1 had no impact in tests at all, so there was no timeout error in agent tests

Reply:
i am ok with 0.9 or 0.8 no problem

Reply:
> #1596 this one. so the thing here if you look at the agent.py. if chain_name is l1 then it continues and actually there is no event monitor right? so that is why adding poll period 2.0 here in conftest for the l1 had no impact in tests at all, so there was no timeout error in agent tests

Ah, I see now, yes, having `l1` in there totally does not make sense, good catch!

Reply:
Actually, here I think we can even configure the global poll period to `0.5` and just remove `poll_period_per_chain`. What do you think?

Reply:
only for tests right? it is ok by me

Reply:
> only for tests right? it is ok by me

Yes, only for tests. Hopefully it makes them a bit faster too.